### ISSUE TITLE
[Errata] Chapter 3: Support Plan Pricing

### ISSUE THREAD

**Issue by mcraghead** [awsccp/awsccp.github.io#27]
**Please describe the error you're reporting**
I think the wrong percentage is used at the Enterprise level for an example in that section. Discussion of an enterprise support plan for an organization spending $750,000/month. Book says it's a 3% "surcharge" at $22,500. However, at the enterprise level, you don't pay 3% until you go over a million dollars.

**Please indicate the chapter or appendix, section, and page number where the error appears (if applicable)**
Chapter 3
Page 35
Paragraph - third

**Please include any other comments or questions**
No additional comments.

**Comment by mcraghead** [comment]
Did anyone look at this or not? There is no response to what was submitted.

**Comment by dbclinton** [comment]
I think this was largely covered by issue #25: 
https://github.com/awsccp/awsccp.github.io/issues/25

**Comment by mcraghead** [comment]
@dbclinton No, this is different. Issue #25 was about the values at the margins straddling two different payment bands under the Enterprise plan (e.g. $150,000 being part of both the 10% band and the 7% band). This (potential) issue that I raise here is not related to the straddling bands. 

In the example in the book that I pointed out, the organization spent **$750,000/month**. That value doesn't straddle any of the bands, so it's not related to #25 . It is firmly in the 5% band. However, the paragraph says that the organization should protect themselves with a **3% support surcharge at $22,500.** That would be the correct amount if $750,000 was in the 3% band, but it's not. The organization would need to spend at least a million dollars to get to a 3% support fee. **The amount they spent is in the 5% band**. The correct answer, given the organization spend each month should be **$37,500**.

Am I missing something or is there an issue with my math? I want to make sure I get this correct.

Thank you.

**Comment by dbclinton** [comment]
Ok. I understand exactly your point now. You're correct: the book's example is wrong (and needs correcting). But your example support charge for $750k of spending ($37,500) is also wrong. The actual cost for support that month would be $52,500.
According to the example on this page: https://aws.amazon.com/premiumsupport/pricing/ the first $150,000 of usage would be billed at 10% - meaning the support cost would be $15,000. The next $350,000 of usage gets a 7% support rate, which would come to $24,500. And support for the final $250,000 of usage would be billed at 5%, which would be $12,500. Meaning, that the total support bill for that month would be $52,000.
This isn't hard math, I have no idea why it keeps getting me all mixed up. :)
David

**Comment by mcraghead** [comment]
Thanks for the update and the correction in my math. My math was wrong because my understanding was wrong. I was computing based on where the monthly spend ended up in the Enterprise plan band. I didn't get that it was cumulative. I'm glad you pointed that out. Otherwise I would have gotten any question related to that incorrect. :-)

Thanks again.

**Comment by thekrol01** [comment]
@dbclinton $15,000 + $24,500 + $12,500=  52000

**Comment by dbclinton** [comment]
@thekrol01 Yup. Right you are. I updated my post. This sad journey never ends, does it? :)

**Comment by thekrol01** [comment]
@dbclinton ha ha, no worries :-)

**Comment by benpiper** [comment]
Adding another report via email:

> The question was as follows: 
> 
> 8) Your company enrolled in the Business Support plan and, through the course of three
> months, consumed $33,000 of AWS services (the consumption was equally divided
> across the months). How much will the support plan cost the company for the full
> three months?: 
> 
> The answer was $33k * 7% = $2310. However as I review the pricing example from AWS, the cost should be calculate as follows: 
> 11k each month: 
> - 10% of first 0 - 10k: 10000 * 10% = 1000
> - 7% charge  from 10k-80k: 1000 * 7% = 70
> - Total each month: 1070. Total price paid 3 months 1070*3 = 3210
>
